js/upload/src/UploadProgress.svelte

```diff
@@ -59,6 +60,7 @@
 		};
 	});
 	onDestroy(() => {
+		// the stream will close itself but is here for perspicuity; remove .close() in 5.0
```

Comment:
the fuck kinda word is this

Reply:
<img width="637" alt="Screenshot 2024-07-11 at 12 45 56" src="https://github.com/gradio-app/gradio/assets/12937446/a90f24d3-577b-4e6b-be74-711cefcdba6a">

are ppl just making words up or something

Reply:
maybe i'm just stupider than i thought i was (and i already thought i was pretty stupid)